### ISSUE TITLE
Card title is capped / limited

### ISSUE THREAD

**Issue by Aesculapius** [nextcloud/deck#88]
When entering a long card title, for example a sentence, the text is first successfully saved - it seems - but later part of it is capped off. I think around 80 characters is the max length? 

I agree that the title should be kept short and put the rest in the description, but maybe a max length of 255 or something like that is more practical.

**Comment by juliushaertl** [comment]
@Aesculapius Thanks for your report. I cannot reproduce this issue on my testing instances. Can you fill out the issue template for details about the database/server setup and maybe post a screenshot where the text is capped?

**Comment by Aesculapius** [comment]
Sure.

### Steps to reproduce
1. Create a new card containing much text
2. Save the card and navigate away from Deck/Nextcloud
3. Come back to the place and notice a lot of characters are missing.

### Expected behaviour
All title text should be kept intact or - when that's really impossible - one should be notices while entering the title text.

### Actual behaviour
Text is capped off.

### Server configuration
### Server configuration
**Operating system**: Linux arc 3.13.0-53-generic #89-Ubuntu SMP Wed May 20 10:34:39 UTC 2015 x86_64

**Web server:** Apache/2.4.7 (Ubuntu) (apache2handler)

**Database:** mysql 5.5.53

**PHP version:** 7.0.1-4+deb.sury.org~trusty+1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, bcmath, calendar, ctype, dba, dom, hash, fileinfo, filter, ftp, gettext, SPL, iconv, mbstring, PDO, session, posix, Reflection, standard, SimpleXML, soap, sockets, Phar, exif, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, zip, mysqlnd, apache2handler, redis, curl, gd, json, mysqli, pdo_mysql, readline, Zend OPcache

**Nextcloud version:** 11.0.2 (stable) - 11.0.2.7

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:** 

**Signing status:**
<details>
	<summary>Signing status</summary>

```
[]
```

</details>

**List of activated apps:**
<details>
	<summary>App list</summary>
```
Enabled:
 - activity: 2.4.1
 - activitylog: 0.0.1
 - admin_notifications: 1.0.0
 - calendar: 1.5.2
 - comments: 1.1.0
 - contacts: 1.5.3
 - dav: 1.1.1
 - deck: 0.1.2
 - federatedfilesharing: 1.1.1
 - federation: 1.1.1
 - files: 1.6.1
 - files_accesscontrol: 1.1.2
 - files_automatedtagging: 1.1.1
 - files_downloadactivity: 1.0.1
 - files_external: 1.1.2
 - files_pdfviewer: 1.0.1
 - files_sharing: 1.1.1
 - files_texteditor: 2.2
 - files_trashbin: 1.1.0
 - files_versions: 1.4.0
 - files_videoplayer: 1.0.0
 - gallery: 16.0.0
 - issuetemplate: 0.2.1
 - logreader: 2.0.0
 - lookup_server_connector: 1.0.0
 - mail: 0.6.2
 - nextcloud_announcements: 1.0
 - notifications: 1.0.1
 - password_policy: 1.1.0
 - provisioning_api: 1.1.0
 - serverinfo: 1.1.1
 - sharebymail: 1.0.1
 - survey_client: 0.1.5
 - systemtags: 1.1.3
 - theming: 1.1.1
 - twofactor_backupcodes: 1.0.0
 - updatenotification: 1.1.1
 - workflowengine: 1.1.1

Disabled:
 - admin_audit
 - encryption
 - external
 - files_retention
 - firstrunwizard
 - templateeditor
 - user_external
 - user_ldap
 - user_saml
```
</details>

**The content of config/config.php:**
<details>
	<summary>Config report</summary>

```
{
    "instanceid": "5141c1f0028db",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "datadirectory": "\/yoda\/cloudstore\/data\/",
    "dbtype": "mysql",
    "version": "11.0.2.7",
    "dbname": "db_owncloud",
    "dbhost": "localhost",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "overwritewebroot": "\/",
    "installed": true,
    "forcessl": true,
    "maintenance": false,
    "trashbin_retention_obligation": "30, auto",
    "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
    "theme": "arc",
    "has_internet_connection": true,
    "check_for_working_webdav": true,
    "check_for_working_htaccess": true,
    "memcache.local": "\\OC\\Memcache\\Redis",
    "filelocking.enabled": true,
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "localhost",
        "port": 6379,
        "timeout": 0,
        "dbindex": 0
    },
    "trusted_domains": [
        "arc.int32.nl"
    ],
    "mail_from_address": "info",
    "mail_smtpmode": "php",
    "mail_domain": "laurensdegroot.nl",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "asset-pipeline.enabled": true,
    "preview_max_scale_factor": 1,
    "forceSSLforSubdomains": true,
    "app.mail.imaplog.enabled": true,
    "loglevel": 2,
    "appstore.experimental.enabled": true
}
```
</details>

**Are you using external storage, if yes which one:** Array
(
    [0] => \OC\Files\Storage\Local
    [1] => \OCA\Files_External\Lib\Storage\FTP
    [2] => \OC\Files\Storage\DAV
    [3] => \OCA\Files_External\Lib\Storage\OwnCloud
    [4] => \OCA\Files_External\Lib\Storage\SFTP
    [5] => \OCA\Files_External\Lib\Storage\AmazonS3
    [6] => \OCA\Files_External\Lib\Storage\Dropbox
    [7] => \OCA\Files_External\Lib\Storage\Google
    [8] => \OCA\Files_External\Lib\Storage\Swift
    [9] => \OCA\Files_External\Lib\Storage\SFTP
    [10] => \OCA\Files_External\Lib\Storage\SMB
    [11] => \OCA\Files_External\Lib\Storage\SMB
)


**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36

Original entered text:
![image](https://cloud.githubusercontent.com/assets/617061/25317289/4d4a36b4-2876-11e7-8596-3f8b4cfd6daf.png)

After signing out and back in again:
![image](https://cloud.githubusercontent.com/assets/617061/25317296/72249f1a-2876-11e7-98ab-57c9cb683b81.png)

**Comment by artemanufrij** [comment]
The database field is limited by 64 chars.

`<field>
    <name>title</name>
    <type>text</type>
    <notnull>true</notnull>
    <length>64</length>
</field>`

I guess the title field of the cards should be limited for 64 chars, too.

**Comment by juliushaertl** [comment]
@artemanufrij Fix is already in #96. Reviews are welcome. :wink:
